package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "developer-portal",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "docs:dev": "vitepress dev docs",
+    "docs:build": "vitepress build docs",
+    "docs:preview": "vitepress preview docs",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
We can remove the test script since we don't have tests.

Reply:
fixed in[16e965e](https://github.com/certifaction/cli/pull/22/commits/16e965edf3023be1f830852759d0c6565151fec7)